### ISSUE TITLE
Backing up the Elastic Runtime MySQL server docs are a bit off

### ISSUE THREAD

**Issue by mikegehard** [pivotal-cf/docs-pcf-install#41]
The existing docs (https://github.com/pivotal-cf/docs-pcf-install/blob/master/backup-restore/backup-pcf.html.md.erb#L220-L242) mention ssh'ing into the MySQL instance and running `mysqldump`. When I tried this, the `mysqldump` command is not found on the VM:

```
vcap@6c9de73f-71c3-4201-8df0-292532bfd5e7:~$ which mysqldump
vcap@6c9de73f-71c3-4201-8df0-292532bfd5e7:~$ exit
```

A workaround is to install mysql on the local machine and just grab the dump from there but the instructions do not mention that. 

This may trip people up if they can not install `mysqldump` locally.

**Comment by menicosia** [comment]
This is an issue that's come up before in the `p-mysql` docs, and the answer is maybe not very satisfying. You'll see that in the `p-mysql` docs _we never tell you to ssh into the mysql VM_.

I think the instructions that the Operator should SSH onto the ERT MySQL VM was introduced by a well meaning person, but is misleading. We don't want Operators accessing the MySQL nodes except in cases of an emergency, and we 100% do not want them installing software. If need be, we could point them to the correct path in `/var/vcap/packages/...` but that's not the right thing to do.

I 100% agree that the docs could be way better about explaining what's going on here, but the gist is:

- Nothing says you need to do backup and restore on a MySQL VM. In fact, the MySQL VMs may not have sufficient disk space for a dump.
- Instead, the implication is that you should run the mysql commands remotely. That's why my docs refer to the MySQL Node IP.
- The instructions SHOULD say something along the lines of, "in order to properly back up the system, spin up a new VM _under your control_, install the mysqldump client of your choice, and ensure that you have sufficient disk space on that VM to contain the dump. Then run these commands..."

I'm totally open to discussion, but that's my point of view for now. What I really want is for users to simply use the automated backup feature we've built for them, but that wasn't available in PCF 1.6.

**Comment by mikegehard** [comment]
@menicosia I'm fine with that explanation. Since the ER tile docs seem to be a bit off, is there a plan to remove that verbiage to reduce this misunderstanding in the future?
